### PR TITLE
Improve integration tests

### PR DESCRIPTION
**Pull request by vishh**

### CODE REVIEW

**Comment by vishh** [comment]
Don't merge this PR until integration test on jenkins passes.

**Comment by vishh** [comment]
Ping @vmarmol @rjnagal. Integration tests are finally working. Time to get this merged. I want this in before other PRs.

File: integration/framework.go [hunk 1]
```diff
@@ -37,11 +40,23 @@ type kubeFramework interface {
 	// Kube client
 	Client() *kube_client.Client
 
-	// Run kubectl.sh command
-	RunKubectlCmd(cmd ...string) (string, error)
+	// Parses and Returns a replication Controller object contained in 'filePath'
+	ParseRc(filePath string) (*api.ReplicationController, error)
```
Comment (rjnagal) [inline integration/framework.go]: nit: Make it ParseRC to match DeleteRC and CreateRC?

File: integration/framework.go [hunk 2]
```diff
@@ -37,11 +40,23 @@ type kubeFramework interface {
...
 
-	// Run kubectl.sh command
-	RunKubecfgCmd(cmd ...string) (string, error)
+	// Parses and Returns a service object contained in 'filePath'
+	ParseService(filePath string) (*api.Service, error)
+
+	// Creates a kube service.
+	CreateService(ns string, service *api.Service) (*api.Service, error)
```
Comment (rjnagal) [inline integration/framework.go]: nit: The signature looks unnatural. We don't need to return *api.Service. I think we can hide the k8 api format and do the simpler version of returning just the error. Same for CreateRC.

Comment (vishh) [inline integration/framework.go]: Makes sense. Done.

File: integration/framework.go (hunk 1 above)
Comment (vishh) [inline integration/framework.go]: Done.

**Comment by vishh** [comment]
@rjnagal @vmarmol Addressed comments. PTAL

**Comment by vmarmol** [comment]
LGTM, waiting on @rjnagal

**Comment by vishh** [comment]
@rjnagal: Jenkins is flaky when it comes to multiple commits per PR. Ignore the jenkins run for this PR and go ahead with the merge. I tested it manually.

**Comment by rjnagal** [comment]
ok :) LGTM

**Pull request merged**